Octokit/Models/Request/IssueUpdate.cs

```diff
@@ -27,10 +27,17 @@ public class IssueUpdate
         /// <remarks>
         /// Only users with push access can set the assignee for new issues. The assignee is silently dropped otherwise.
         /// </remarks>
-        [SerializeNull]
```

Comment:
@shiftkey you might want to take a look at this too.  It's another "tweak" id made when pushing some changes over to @maddin2016 .  I think it's because now with the multiple assignees support, this field can actually be `null`, but we dont want to send it in as such or else it wiped out all the other assignees.  But I also may not have been aware of historic things similar to how I may have messed up the `Labels` handling currently being discussed!  So if you could check this out too and have a think about it etc to make sure I've done the right thing here?